docs/UserGuide.adoc

```diff
@@ -273,37 +273,123 @@ Examples:
 
 * `set-theme <FILE_NAME>.xml`
 
-=== Add a new administrator account: `addAdmin` `[coming in v2.0]`
+// tag::admincommands[]
+=== Add a new administrator account: `addAdmin`
 
 Create and add a new adminstrator account. +
-Format: `addAdmin user\<USERNAME> pass\<PASSWORD> n\<NAME> ptp\<PATH_TO_PIC> salary\<SALARY> date\<EMPLOYMENTDATE>`
+Format: `addAdmin OPTION/INPUT...` +
 
-Examples:
+Input fields:
+
+* Username : `user/<USERNAME>`
```

Comment:
If there is a certain restriction for inputs, I think its a good idea to include them here.
Its better if a user can see such restriction here rather than seeing the error message in the application.

Reply:
restriction that are implemented already have been added.